sdcm/cluster.py

```diff
@@ -2786,6 +2791,21 @@ def _node_setup(self, node, seed_address):
         scylla_yaml_contents = scylla_yaml_contents.replace("cluster_name: 'Test Cluster'",
                                                             "cluster_name: '{0}'".format(self.name))
 
+        if node.is_addition:
```

Comment:
Apart from the fact that this would sound more natural if the attribute was named `added`, this should be OK.